testbed/testbed/receivers.go

```diff
@@ -79,12 +79,7 @@ func (bor *BaseOTLPDataReceiver) Start(tc consumer.Traces, mc consumer.Metrics,
 		return err
 	}
 
-	if err = bor.traceReceiver.Start(context.Background(), componenttest.NewNopHost()); err != nil {
-		return err
-	}
-	if err = bor.metricsReceiver.Start(context.Background(), componenttest.NewNopHost()); err != nil {
-		return err
-	}
+	// we reuse the receiver across signals. Starting the log receiver starts the metrics and traces receiver.
```

Comment:
This is confusing, do we need the three factory calls above then?

Reply:
you need them to set the logs, metrics and traces consumer.